@here/harp-datasource-protocol/lib/TechniqueParams.ts

```diff
@@ -691,12 +691,32 @@ export interface MarkerTechniqueParams extends BaseTechniqueParams {
     wrappingMode?: DynamicProperty<"None" | "Character" | "Word">;
     /**
      * Text position regarding the baseline.
+     *
```

Comment:
See the documentation for xOffset, yOffset. It's wrongly stating a hardcoded offset direction:
>  * Horizontal offset (to the right) in screen pixels.
> * Vertical offset (up) in screen pixels.